### DISCUSSION TITLE
Text Decoration limited to 500 spell check errors? How in increase?

### DISCUSSION THREAD

**Discussion by ranger81** [streetsidesoftware/vscode-spell-checker#3842] [Q&A] [answered]
Hi,
I have a super large document opened in VS Code and the spell checker is reporting 1177 incorrect words in this document. 
However, the blue curly underline (Text Decoration for incorrect words) in my document seems to be limited to the first 500 incorrectly spelled words. So far I could not find a corresponding setting to increase that limit. What did I miss?

**Comment by Jason3S** [comment, marked as answer]
@ranger81,

Since VS Code limits the number of issues shown in a document, it is necessary to use custom decorations to draw the `~~~~`.

To do this, set: `cSpell.useCustomDecorations` to `true`:
<img width="873" alt="image" src="https://github.com/user-attachments/assets/a304df82-4e7e-4663-9162-c30b67a111bb">

It should be able to draw all the issues.

**Reply by ranger81** [reply]
Thanks, works perfectly.
